staging/src/k8s.io/apiserver/pkg/admission/attributes.go

```diff
@@ -140,6 +145,70 @@ func (record *attributesRecord) AddAnnotation(key, value string) error {
 	return nil
 }
 
+func (record *attributesRecord) GetReinvocationContext() ReinvocationContext {
+	return record.reinvocationContext
+}
+
+func newReinvocationContext() *reinvocationContext {
+	return &reinvocationContext{previouslyInvokedRevocableWebhooks: sets.NewString(), reinvokeWebhooks: sets.NewString()}
+}
+
+type reinvocationContext struct {
+	// isReinvoke is true when admission plugins are being reinvoked
+	isReinvoke bool
+	// lastOutput holds the result of the last webhook admission plugin call
+	lastOutput runtime.Object
```

Comment:
lastWebhookOutput might be clearer, if it is only about webhooks.

Reply:
Thanks, that is clearer.